dll/cpl/intl/lang/ro-RO.rc

```diff
@@ -39,83 +39,83 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-    CHECKBOX "Instalează fișierele pentru script-uri comple&xe și pentru limbile cu scriere inversă", IDC_INST_FILES_FOR_RTOL_LANG, 12, 100, 215, 22, BS_MULTILINE | BS_TOP
-    CHECKBOX "Instalează fișierele pentru limbile din &estul Asiei", IDC_INST_FILES_FOR_ASIAN, 12, 119, 180, 22, BS_MULTILINE | BS_TOP
+    GROUPBOX "Suport pentru limbi suplimentare", -1, 5, 62, 234, 82
+    LTEXT "Majoritatea limbilor sunt instalate implicit. Pentru a instala limbi suplimentare, selectaţi caseta corespunzătoare.", -1, 12, 72, 220, 25
+    CHECKBOX "I&nstalare fişiere pentru script-uri complexe şi pentru limbi cu scriere de la dreapta la stânga (inclusiv Thai)", IDC_INST_FILES_FOR_RTOL_LANG, 12, 100, 215, 22, BS_MULTILINE | BS_TOP
+    CHECKBOX "In&stalare fişiere pentru limbile est-asiatice", IDC_INST_FILES_FOR_ASIAN, 12, 119, 180, 22, BS_MULTILINE | BS_TOP
 END
 
 IDD_ADVANCEDPAGE DIALOGEX 0, 0, 246, 230
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Avansate"
+CAPTION "Complex"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Limba pentru programe ne-Unicode", -1, 5, 5, 234, 90
+    GROUPBOX "Limbi pentru programele non-Unicode", -1, 5, 5, 234, 90
     COMBOBOX IDC_LANGUAGE_COMBO, 14, 75, 217, 160, CBS_DROPDOWNLIST | CBS_NOINTEGRALHEIGHT | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP | CBS_SORT
-    LTEXT "Această opțiune specifică limba de afișare pentru programele care nu utilizează Unicode în textele din meniuri și ferestrele de dialog. Opțiunea nu afectează aplicațiile Unicode, dar va afecta toți utilizatorii din sistem.", -1, 14, 18, 223, 33
-    LTEXT "Alegeți limba ce corespunde cu limba aplicațiilor ne-Unicode pe care doriți să le utilizați:", -1, 14, 55, 223, 18
-    GROUPBOX "Tabele de conversie cod-pagină", -1, 5, 101, 234, 88
+    LTEXT "Setările acestui sistem permit programelor non-Unicode să afişeze meniuri şi dialoguri în limba lor de origine. Acest fapt nu afectează programele Unicode, dar se aplică pentru toţi utilizatorii acestui computer.", -1, 14, 18, 223, 33
+    LTEXT "&Selectaţi o limbă care să corespundă versiunii de limbă a programelor non-Unicode pe care le veţi utiliza:", -1, 14, 55, 223, 18
+    GROUPBOX "Tabele de &conversie a paginilor de cod", -1, 5, 101, 234, 88
     CONTROL "", IDC_CONV_TABLES, "SysListView32", LVS_REPORT | LVS_SORTASCENDING | LVS_NOCOLUMNHEADER | WS_BORDER | WS_TABSTOP, 14, 114, 217, 70
-    GROUPBOX "Opțiuni implicite pentru conturi de utilizator", -1, 5, 193, 234, 32
-    CHECKBOX "Apli&că aceste opțiuni atât pentru contul de utilizator curent cât și pentru cel implicit", IDC_APPLY_CUR_USER_DEF_PROFILE, 12, 201, 220, 22, BS_MULTILINE | BS_TOP
+    GROUPBOX "Setările implicite pentru conturile de utilizator", -1, 5, 193, 234, 32
+    CHECKBOX "Se aplică toate setările la contul utilizatorului curent şi la profilul utilizator &implicit", IDC_APPLY_CUR_USER_DEF_PROFILE, 12, 201, 220, 22, BS_MULTILINE | BS_TOP
```

Comment:
Are you sure that romanian 2k3 does use &i here? MS does typically not recommend that. If you can prove it with a pic, then it is fine ofc.

Reply:
Yes. MS did that here. Here is the proof:

![2024-08-17 00_46_04-Resource Hacker - intl cpl mui](https://github.com/user-attachments/assets/7f2b7e7a-5e59-4bf1-ab37-2592058582f1)